itou/siaes/tests/tests_import_siae_command.py

```diff
@@ -210,3 +212,13 @@ def test_activate_your_account_email_for_a_siae_without_members_but_with_auth_em
             ],
             [mail.subject for mail in mail.outbox],
         )
+
+
+@override_settings(METABASE_HASH_SALT="foobar2000")
+def test_hashed_approval_number():
+    df = pd.DataFrame(data={"salarie_agrement": ["999992012369", None, ""]})
+    anonymize_fluxiae_df(df)
+    assert df.hash_numéro_pass_iae[0] == "314b2d285803a46c89e09ba9ad4e23a52f2e823ad28343cdff15be0cb03fee4a"
+    assert df.hash_numéro_pass_iae[1] == "8e728c4578281ea0b6a7817e50a0f6d50c995c27f02dd359d67427ac3d86e019"
+    assert df.hash_numéro_pass_iae[2] == "6cc868860cee823f0ffe0b3498bb4ebda51baa1b7858e2022f6590b0bd86c31c"
+    assert "salarie_agrement" not in df
```

Comment:
Pinaillage : c'est un peu surprenant que ce test qui n'a a priori rien à voir avec l'import SIAE se retrouve dans `itou/siaes/tests/tests_import_siae_command.py` mais ça s'explique.

Historiquement il y a un peu de logique commune FluxIAE entre l'import SIAE et l'export Metabase hebdo, pour la raison suivante : l'import SIAE utilise deux vues FluxIAE, la vue Structure et la vue AF. Et ce code se retrouve dans `_import_siae/utils.py` par exemple `get_fluxiae_df` et `anonymize_fluxiae_df`.

Je m'y suis mal pris et avec le recul je me dis qu'il serait plus élégant que le code FluxIAE commun à l'import SIAE et à Metabase soit dans un espace propre séparé, par exemple `itou/utils/fluxiae.py`.

C'est pas bloquant hein.

Reply:
Yes, je me suis dit la meme chose. Je serais content de faire une petite refonte en ce sens, ce n'est pas la premiere fois que je suis dérangé par ce code "en commun" qui en vrai mériterait sa propre logique séparée plutot que d'etre mutuellement dépendant. Noté dans un coin de tête.